### PR TITLE
Fixed homebrew install instructions

### PR DESCRIPTION
**Pull request by perryprog**
In the homebrew updates since macOS Sierra, brew automatically updates.

### CODE REVIEW

File: app/views/docs/_install_homebrew.html.md [hunk 1]
```diff
@@ -1 +1 @@
-    brew update && brew install asciinema
+brew install asciinema
```
Comment (sickill) [inline app/views/docs/_install_homebrew.html.md]: Well, it's still missing indentation ;)

Comment (perryprog) [inline app/views/docs/_install_homebrew.html.md]: Ah, I see. I think if you merge now, it should come through without the indention. As in, you edited the commit.

**Comment by sickill** [comment]
This second commit "Test update" seems to not have any changes.... 
I think it would be best if you opened a fresh PR with single commit for this change.

**Comment by perryprog** [comment]
K

**Comment by perryprog** [comment]
Oh - there was already a tab in the file. Should I delete that or leave it be?

**Comment by sickill** [comment]
It should stay :)

**Pull request closed without merging**